gallery/how_to/work_with_pytorch/using_optimized_torch.py

```diff
@@ -0,0 +1,176 @@
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Compile PyTorch Models
+======================
+**Author**: 
+`Yaoda Zhou <https://github.com/juda>`_,
+`Masahiro Masuda <https://github.com/masahi>`_
+
+This article is an introductory tutorial to optimize PyTorch models by using `tvm.contrib.torch.optimize_torch`.
+To follow this tutorial, PyTorch, as well as TorchVision, should be installed.
+"""
+
+# sphinx_gallery_start_ignore
+from tvm import testing
+
+testing.utils.install_request_hook(depth=3)
+# sphinx_gallery_end_ignore
+
+# Import PyTorch
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+# Import library for profiling
+import torch.utils.benchmark as benchmark
+from torchvision.models import resnet18
+
+# Import `optimize_torch` function
+from tvm.contrib.torch import optimize_torch
+from tvm.meta_schedule import TuneConfig
+
+######################################################################
+# Define a simple module written by PyTorch
+# ------------------------------
+
+
+class SimpleModel(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.conv1 = nn.Conv2d(1, 20, 5)
+        self.conv2 = nn.Conv2d(20, 20, 5)
+
+    def forward(self, x):
+        x = F.relu(self.conv1(x))
+        return F.relu(self.conv2(x))
+
+
+######################################################################
+# Optimized SimpleModel by TVM MetaSchedule
+# ------------------------------
+# We provide a `optimize_torch` function, which has the similar usage as `torch.jit.trace`.
+# The PyTorch model to optimize, along with its example input, are provided by users.
+# We can optimized the PyTorch's module by calling the `optimized_torch` method in default setting.
+# Without providing extra information, the model will be tuned for CPU.
+
+simple_model = SimpleModel()
+example_input = torch.randn(20, 1, 10, 10)
+model_optimized_by_tvm = optimize_torch(simple_model, example_input)
+
+######################################################################
+# Save/Load module
+# ------------------------------
+# We can save and load our tuned module like the standard `nn.module`.
+
+# Let us run our tuned module.
+ret1 = model_optimized_by_tvm(example_input)
+
+torch.save(model_optimized_by_tvm, "model_optimized.pt")
+model_loaded = torch.load("model_optimized.pt")
+
+# We load the module and run it again.
+ret2 = model_loaded(example_input)
+
+# We show that the results from original SimpleModel,
```

Comment:
"We show that the results from the original PyTorch model and the model tuned by TVM are the same, as expected."

No need to verify the result of the "loaded model". Note that the term "loaded model" makes no sense.